### PR TITLE
docs: update adapter-node.md

### PR DESCRIPTION
**Pull request by BCsabaEngine**
Fix the ```npm ci``` command line for actual version
Extend ORIGIN samples for local development

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [x] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs
- [x] This message body should clearly illustrate what problems it solves.
- [ ] Ideally, include a test that fails without this PR but passes with it.

### Tests
- [ ] Run the tests with `pnpm test` and lint the project with `pnpm lint` and `pnpm check`

### Changesets
- [ ] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpm changeset` and following the prompts. Changesets that add features should be `minor` and those that fix bugs should be `patch`. Please prefix changeset messages with `feat:`, `fix:`, or `chore:`.

### CODE REVIEW

File: documentation/docs/25-build-and-deploy/40-adapter-node.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ export default {
 
 First, build your app with `npm run build`. This will create the production server in the output directory specified in the adapter options, defaulting to `build`.
 
-You will need the output directory, the project's `package.json`, and the production dependencies in `node_modules` to run the application. Production dependencies can be generated with `npm ci --prod` (you can skip this step if your app doesn't have any dependencies). You can then start your app with this command:
+You will need the output directory, the project's `package.json`, and the production dependencies in `node_modules` to run the application. Production dependencies can be generated (after package-lock.json copied here too) with `npm ci --omit dev` (you can skip this step if your app doesn't have any dependencies). You can then start your app with this command:
```
Comment (benmccann) [inline documentation/docs/25-build-and-deploy/40-adapter-node.md:27]: two sets of parentheses in one sentence feels like a bit much. we should put the file name in backticks
```suggestion
You will need the output directory, the project's `package.json`, and the production dependencies in `node_modules` to run the application. Production dependencies can be generated by copying the `package.json` and `package-lock.json` and then running `npm ci --omit dev` (you can skip this step if your app doesn't have any dependencies). You can then start your app with this command:
```

**Comment by benmccann** [comment]
A note to other reviewers as to the difference here: if a package is listed in both `dependencies` and `devDependencies` the updated command will install it whereas the prior command will not. I suppose this seems like a reasonable change

**Comment by Rich-Harris** [comment]
> the updated command will install it whereas the prior command will not

wait `npm ci --prod` will omit things that are in `dependencies` and `devDependencies` but `npm ci --omit-dev` won't? that seems... surprising

File: documentation/docs/25-build-and-deploy/40-adapter-node.md [hunk 2]
```diff
@@ -63,6 +63,8 @@ HTTP doesn't give SvelteKit a reliable way to know the URL that is currently bei
 
 ```
 ORIGIN=https://my.site node build
+ORIGIN=http://localhost:3000 node build
+ORIGIN=http://127.0.0.1:4000 node build
```
Comment (Rich-Harris) [inline documentation/docs/25-build-and-deploy/40-adapter-node.md:67]:
```suggestion
ORIGIN=https://my.site node build

# or e.g. for local previewing and testing
ORIGIN=http://localhost:3000 node build
```

**Comment by benmccann** [comment]
> wait npm ci --prod will omit things that are in dependencies and devDependencies but npm ci --omit-dev won't? that seems... surprising

According to ChatGPT, which is what I asked when I was confused by the motivation for this PR. Not sure how much you trust it. It's told me things I know to be blatantly wrong before, so probably wouldn't hurt to verify :grin:

**Comment by Rich-Harris** [comment]
The docs don't mention a `--prod` flag at all, no idea where that came from

https://docs.npmjs.com/cli/v9/commands/npm-ci

**Comment by benmccann** [comment]
Ah. `npm install` has a `--production` flag. Probably this used to be `npm install` and no one realized that npm was stupid enough to use completely different syntax when changing to `npm ci`

**Review by benmccann — APPROVED** [review]

**Review by Rich-Harris — APPROVED** [review]

**Comment by Rich-Harris** [comment]
thanks!

**Pull request merged**